nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,31 @@
+const { request, response } = require("express");
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("../data/reservations.json");
+//	Respond with the json for all reservations
+router.get("/", async (request, response) => {
+  try {
+    console.log("in /api/reservations");
+    response.statusCode = 200;
+    response.send(reservations);
+  } catch (error) {
+    throw error;
+  }
+});
+//Respond with the json for the reservation with the corresponding id
+router.get("/:id", async (request, response) => {
+  try {
+    const id = parseInt(request.params.id);
+    console.log(id);
+    const reservationID = reservations.find((reservation) => reservation.id === id);
```

Comment:
I like that are making sure the id sent in is a number and are parsing it, well done :)
Regarding the name 'reservationID', it's not 100% accurate since it does not contain just the id, but the whole reservation object, so maybe change it to something like 'reservation'?